core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -505,6 +507,36 @@ public int onStartCommand(Intent intent, int flags, int startId) {
         return Service.START_NOT_STICKY;
     }
 
+    private void skipIntro(Playable playable) {
+
+        int skipIntro = 0;
+        if (! (playable instanceof FeedMedia)) { return; }
+        if (playable.getLastPlayedTime() > 0) { return; }
```

Comment:
There is a checkstyle file in config/checkstyle/ that I want to enforce for new code. Because the existing code is quite inconsistent, CI only checks a small fraction of things. In the past months, I did the checks manually and commented on GitHub. In #3978, I enabled checking the diff on CI.

I think checkstyle will complain about this line. Have not checked the rest of the code. Just that you already know before I merge #3978 and it suddenly complains.

Reply:
not sure what it should be instead?
```
if (playable.getLastPlayedTime() > 0) { return; }
```

Reply:
@ByteHamster could you check the checkstyle, it looks like the error is not related to my changes or the reporting is not super clear